index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+    <head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio project</title>    
+  <link rel="stylesheet" href="styles.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+    <body>
+        <header>
+            <nav>
+                <ul>
+              <li><a class="logolink" href="">My Logo</a></li>
+              <li class="leftli"><a href=""><img class="imageone" src="hammenu.png" alt="ham"></a></li>
+                </ul>
+            </nav>
+        </header>
+        <div class="headline">
```

Comment:
- [ ] [OPTIONAL] It would be nice and semantic if you change the `div` tag to a `section` tag because the container of the section should be a `section` tag 👍🏼